spec/functional/resource/group_spec.rb

```diff
@@ -267,14 +267,14 @@ def create_group
     end
 
     describe "when removing members" do
-      it "raises an error for a non well-formed domain name" do
+      it "does not raise an error for a non well-formed domain name" do
```

Comment:
I guess we could special case malformed vs. non-existent, however at the API level it doesn't really care and has the same result (raise error).  It seemed simplest / safest to just warn and continue when checking membership for removal.